### ISSUE TITLE
error in render_highquality 

### ISSUE THREAD

**Issue by SharadShingade** [tylermorganwall/rayshader#255]
hi, I am continuously getting following error whenever i call function render_highquality() . I reinstalled rayshader & rayrender from git still throwing same error.  I also tried on different R version and also different machines but still same error. Has anyone have some suggestion to solve this. Thanks

```
Error in rayrender::obj_model(cache_filename, x = -bbox_center[1], y = -bbox_center[2],  : 
  unused argument (texture = TRUE)
```

**Comment by maniatisn** [comment]
hey, I opened the same issue a few hours before you, I'm having the same problem.

**Comment by canaytore** [comment]
this error no longer appears with the newest versions. you may like to install the latest version of rayshader (==0.32.7) via devtools using `devtools::install_github("tylermorganwall/rayshader")`

**Comment by tylermorganwall** [comment]
Yes, update rayshader to the latest version on github and this should go away.

```r
remotes::install_github("tylermorganwall/rayshader")
```

**Comment by SharadShingade** [comment]
Thanks. Issue resolved with downgrading R version (4.2.0 ) and installing rayshader & rayrender via github.
